### DISCUSSION TITLE
When I use Dalle3 in the conversation window to request the generation of two different images at once

### DISCUSSION THREAD

**Discussion by vislamu** [danny-avila/LibreChat#3730] [Troubleshooting]
### What happened?

When I use Dalle3 in the conversation window to request the generation of two different images at once, the interface displays an error, and the API's deduction is correct

### Steps to Reproduce

1、endpoint select "plugins", model select "gpt-4o", tools select "dalle3";
2、 to request the generation of two different images at once;
![error](https://github.com/user-attachments/assets/74a23800-13c2-4ec1-b4e9-8dd275677708)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Duplicate: https://github.com/danny-avila/LibreChat/discussions/3320
